static/panes/compiler.js

```diff
@@ -1430,9 +1430,35 @@ Compiler.prototype.postCompilationResult = function (request, result, wasCmake)
 
     if (result.bbcdiskimage) {
         this.emulateBbcDisk(result.bbcdiskimage);
+    } else if (result.speccytape) {
+        this.emulateSpeccyTape(result.speccytape);
     }
 };
 
+Compiler.prototype.emulateSpeccyTape = function (image) {
+    var dialog = $('#jsspeccyemu');
+
+    this.alertSystem.notify(
+        'Click ' +
+            '<a target="_blank" id="jsspeccy_emulink" style="cursor:pointer;" click="javascript:;">here</a>' +
+            ' to emulate',
+        {
+            group: 'emulation',
+            collapseSimilar: true,
+            dismissTime: 10000,
+            onBeforeShow: function (elem) {
+                elem.find('#jsspeccy_emulink').on('click', function () {
+                    dialog.modal();
+
+                    var emuwindow = dialog.find('#speccyemuframe')[0].contentWindow;
+                    var tmstr = Date.now();
+                    emuwindow.location = 'https://speccy.compiler-explorer.com/?' + tmstr + '#b64tape=' + image;
```

Comment:
Maybe this should be something on `ce-cdn.net` instead, as we don't need the cookies, etc

Reply:
The easy route is probably to put it in the S3 bucket of static.ce-cdn.net in a subdirectory. The only problem; there are way too many files in that bucket, maybe we should clean it up somehow by deleting really old versions?

Reply:
Works for me. We can add a policy to delete files older than a year or two. But really it's not a huge problem I don't think. Ideally we wouldn't have them I mean but it's not like we lost the directory often.